examples/erc1155/tests/erc1155.rs

```diff
@@ -1568,3 +1565,291 @@ async fn error_when_insufficient_balance_safe_batch_transfer_from(
 
     Ok(())
 }
+
+#[e2e::test]
+async fn burns(alice: Account) -> eyre::Result<()> {
+    let contract_addr = alice.as_deployer().deploy().await?.address()?;
+    let contract = Erc1155::new(contract_addr, &alice.wallet);
+
+    let alice_addr = alice.address();
+    let token_ids = random_token_ids(1);
+    let values = random_values(1);
+
+    let _ = watch!(contract.mint(
```

Comment:
I thought about updating the "happy flow" mint-related e2e tests to act as a setup for "happy flow" burn tests (first test mint, then test burn), but opted for the currently conventional approach.